projects/valtimo/components/src/lib/components/modal/modal.component.ts

```diff
@@ -44,8 +44,12 @@ export class ModalComponent implements AfterViewInit, OnInit, OnDestroy {
 
   @ViewChild('scrollModal') scrollModal: ElementRef<HTMLDivElement>;
 
+  readonly modalShowing$ = new BehaviorSubject<boolean>(false);
```

Comment:
private

Reply:
This does not work, since modalShowing$ has to be accesible from a viewChild 
![image](https://user-images.githubusercontent.com/79840403/201633403-cf097751-3c9c-4de4-b0be-cc651a31985a.png)